### PR TITLE
Add list contacts from event feature

### PR DESCRIPTION
**Pull request by weilin1202**
Add the list contacts from event feature 
- The command for this feature is listevcontact followed by an event index 
- All the persons whose event set contains the specified event in the address book will be listed, not just the currently filtered list of persons shown on the UI
- Only the list of persons featured on the UI will be updated, the list of events on the UI will not be modified

### CODE REVIEW

File: src/test/java/seedu/address/model/person/EventSetContainsEventPredicateTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    @Test
+    public void test_personEventSetContainsEvent_returnsTrue() {
+        EventSetContainsEventPredicate predicate = new EventSetContainsEventPredicate(WEDDING_DINNER);
+        assertTrue(predicate.test(new PersonBuilder().withEventSet(WEDDING_DINNER).build()));
+
+    }
+
+    @Test
+    public void test_nameDoesNotContainKeywords_returnsFalse() {
```
Comment (venuslimm) [inline src/test/java/seedu/address/model/person/EventSetContainsEventPredicateTest.java:48]: Does 'nameDoesNotContainKeywords' mean the Person's EventSet does not contain the given target event?

**Review by venuslimm — COMMENTED** [review]
The code looks good! I have left a comment to clarify something unclear.

File: src/test/java/seedu/address/model/person/EventSetContainsEventPredicateTest.java (hunk 1 above)
Comment (weilin1202) [reply src/test/java/seedu/address/model/person/EventSetContainsEventPredicateTest.java]: Yup! Edited, thanks!

**Review by venuslimm — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/ListEvContactCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import seedu.address.logic.commands.ListEvContactCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ListEvContactCommand object
+ */
+public class ListEvContactCommandParser implements Parser<ListEvContactCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the ListEvContactCommand
+     * and returns a ListEvContact object for execution.
```
Comment (inas0ng) [inline src/main/java/seedu/address/logic/parser/ListEvContactCommandParser.java:15]:
```suggestion
     * and returns a ListEvContactCommand object for execution.
```

**Review by inas0ng — CHANGES_REQUESTED** [review]
looks good to merge! j a minor javadocs typo to change :)

File: src/main/java/seedu/address/logic/parser/ListEvContactCommandParser.java (hunk 2 above)
Comment (weilin1202) [reply src/main/java/seedu/address/logic/parser/ListEvContactCommandParser.java]: Edited, thankss

**Review by inas0ng — APPROVED** [review]
LGTM!

**Pull request merged**